memtar.gemspec

```diff
@@ -12,14 +12,17 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/conjurinc/memtar"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  #spec.files         = `git ls-files -z`.split("\x0")
```

Comment:
do we still need the comment

Reply:
I forgot to uncomment this after testing.  It's the when clause to make it only publish when a release is needed.